demos/Demo AreaLight.cpp

```diff
@@ -11,14 +11,14 @@ int main()
 

```

Comment:
I think this file is not affected by the DoF and should be removed from the pull request

Reply:
yes, I will revert it back to original